src/placeos-driver/interface/electrical_relay.cr

```diff
@@ -0,0 +1,19 @@
+abstract class PlaceOS::Driver; end
+
+module PlaceOS::Driver::Interface::ElectricalRelay
+  abstract def relay(state : Bool, index : Int32 = 0, **options)
+
+  def pulse(period : Float64 = 1.0, index : Int32 = 0, times : Int32 = 1, initial_state = false)
```

Comment:
It may be better to specify period in milliseconds, or alternatively support both (float = seconds, int = ms).